pilot/pkg/serviceregistry/kube/controller/namespacecontroller.go

```diff
@@ -70,6 +75,11 @@ func NewNamespaceController(kubeClient kube.Client, caBundleWatcher *keycertbund
 		controllers.WithReconciler(c.reconcileCACert),
 		controllers.WithMaxAttempts(maxRetries))
 
+	c.ignoredNamespaces = inject.IgnoredNamespaces.Copy()
+	if platform.IsOpenShift() && features.EnableAmbientControllers {
+		c.ignoredNamespaces.Delete(constants.KubeSystemNamespace)
+	}
```

Comment:
Can you split this out from the profile change?

Reply:
Yes, sure.

Reply:
https://github.com/istio/istio/pull/49129

Reply:
Done